<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerWidget</class>
 <widget class="QFrame" name="VideoPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="camFrameVLayout">
     <item>
      <layout class="QVBoxLayout" name="viewTypeSelectorLayout"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="settingsHLayout">
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="speedSelector">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>0.25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.75</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxLive">
         <property name="text">
          <string>Live</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="navigatorLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
